packages/shared/src/components/post/BasePostContent.tsx

```diff
@@ -51,6 +59,11 @@ export function BasePostContent({
       {customNavigation ?? (
         <PostNavigation {...navigationProps} className={className.navigation} />
       )}
+      {withPillTypes.includes(post?.type) && (
```

Comment:
Should this change be dependant on the experiment?

Reply:
This actually exists already on the pages where it is needed. Will remove this. Thank you!